docs/reference/snapshot-restore/apis/delete-snapshot-api.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[delete-snapshot-api]]
+=== Delete snapshot API
+++++
+<titleabbrev>Delete snapshot</titleabbrev>
+++++
+
+Deletes a <<snapshot-restore,snapshot>> of a cluster or specified data streams and
+indices.
+
+////
+[source,console]
+----
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+----
+// TESTSETUP
+////
+
+[source,console]
+-----------------------------------
+DELETE /_snapshot/my_repository/my_snapshot
+-----------------------------------
+// TEST[s/my_snapshot/my_snapshot]
```

Comment:
I don't think this `// TEST` comment does anything. You can probably remove it.
```suggestion
```

Reply:
Funny you say that, because I _just_ removed that TEST 😆